README.md

```diff
@@ -54,13 +54,11 @@ You should install Docker on you system as requirement **OR** just using the **G
 
 1- Clone the repository.
 
-2- Change the branch to `oc-opencloudmesh-testing`.
+2- Run `./scripts/clean.sh`.
 
-3- Run `./scripts/clean.sh`.
```

Comment:
but without the ./scripts/clean.sh we are getting the "testnet network is not defined" error or some thing like that. least astonishment breaches but.

Reply:
Look at line 57 :)

This step wasn't removed, it was just relabeled from `3-` to `2-`